util/liblog/liblog.go

```diff
@@ -13,6 +13,9 @@ type Logr interface {
 func FromSDKLogger(l log.Logger) Logr {
 	return &lgwr{l: l}
 }
+func (l *lgwr) Impl() interface{} {
+	return l
+}
```

Comment:
@aleem1314 Where does this come from? I think we can remove this.

Reply:
In v0.50, `Impl()` method added to the `Logger` interface. So implementation required for each type. [Ref](https://github.com/cosmos/cosmos-sdk/blob/main/log/logger.go#L57)

Reply:
Well this is ugly AF ... Consider resolved.